ext/xmlreader/tests/003-get-errors.phpt

```diff
@@ -9,7 +9,7 @@ Mark Baker mark@lange.demon.co.uk at the PHPNW2017 Conference for PHP Testfest 2
 // Set up test data in a new file
 $xmlstring = '<?xml version="1.0" encoding="UTF-8"?>
 <books><book num="1" idx="2">book1</book></books>';
-$filename = dirname(__FILE__) . '/_014.xml';
+$filename = __DIR__ . '/_003-1.xml';
```

Comment:
I'd prefer it to give these the same name as the test, i.e. `_003-get-errors.xml` or even just `003-get-errors.xml`. That way we know for sure that these files will never clash.

Reply:
The names of the XML files are updated according to your recommendations.